spec/namespaces/cluster.yaml

```diff
@@ -1380,6 +1413,27 @@ components:
       schema:
         $ref: '../schemas/_common.yaml#/components/schemas/Duration'
       style: form
+    cluster.stats::path.index_metric:
+      in: path
+      name: index_metric
+      description: Limit the information returned for indices metric to the specific index metrics. It can be used only if indices (or all) metric is specified.
```

Comment:
Vale is correct here, please fix, indexes is the standard in the documentation.

Reply:
Fixed.